### DISCUSSION TITLE
is there option to send real user-agent detail along with the DOH-DNS-Request

### DISCUSSION THREAD

**Discussion by linuxserver2016** [DNSCrypt/dnscrypt-proxy#2528] [Q&A]
### Requirement: I want to differentiate the User-Agent Details in DOH-Request but nginx access log getting only "dnscrypt-proxy"

**Example:**

`192.168.2.10 - -[15/Nov/2023:17:38:44 +0530] "POST /o4k7wkkj?body_hash=8871c3e628d3298a0546ca2b992e098db903fff6b126e716b07528d03c242870 HTTP/1.1" 200 168 "-" "dnscrypt-proxy"`

**Comment by lifenjoiner** [comment]
Negative, just solid `dnscrypt-proxy` as you see.
